conda_forge_tick/deploy.py

```diff
@@ -148,22 +155,17 @@ def _deploy_batch(*, files_to_add, batch, n_added, max_per_batch=200):
                     pass
 
                 print(">>>>>>>>>>>> git push try", flush=True)
-                with sensitive_env() as env:
-                    status = run_command_hiding_token(
-                        [
-                            "git",
-                            "push",
-                            "https://{token}@github.com/{deploy_repo}.git".format(
-                                token=env.get("BOT_TOKEN", ""),
-                                deploy_repo="regro/cf-graph-countyfair",
-                            ),
-                            "master",
-                        ],
-                        token=env.get("BOT_TOKEN", ""),
-                    )
-                if status != 0:
-                    print(">>>>>>>>>>>> git push failed", flush=True)
-
+            status = run_command_hiding_token(
+                [
+                    "git",
+                    "push",
+                    f"https://{get_bot_token()}@github.com/{GRAPH_REPO}.git",
+                    "master",
```

Comment:
Seeing `master` here I wonder whether we also need a `GRAPH_REPO_DEFAULT_BRANCH` setting.

Reply:
Yes that'd be great. I have not switched over that repo to main for exactly this reason.

Reply:
Since I don't see a use case for configuring this via an environment variable, I added this as a constant to settings.py here: 7535b95f743a6fa500adc1081303fbc4fb420148